src/test/regress/expected/validate_constraint.out

```diff
@@ -0,0 +1,256 @@
+ERROR:  current transaction is aborted, commands ignored until end of transaction block
+ROLLBACK;
+-- drop table for next tests
+DROP TABLE constrained_table;
+DROP TABLE referenced_table;
+CREATE TABLE constrained_table (
+  id                 int,
+  constrained_column int
+);
+SELECT create_distributed_table('constrained_table', 'id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+INSERT INTO constrained_table
+SELECT x, x
+from generate_series(1, 1000) as f (x);
+-- a constraint that can be validated
+ALTER TABLE constrained_table
+  ADD CONSTRAINT validatable_constraint CHECK (constrained_column < 10000) NOT VALID;
+ALTER TABLE constrained_table
+  VALIDATE CONSTRAINT validatable_constraint;
+-- a constraint that can not be validated
+ALTER TABLE constrained_table
+  ADD CONSTRAINT failing_constraint CHECK (constrained_column > 100) NOT VALID;
+ALTER TABLE constrained_table
+  VALIDATE CONSTRAINT failing_constraint;
+ERROR:  check constraint "failing_constraint_8000009" is violated by some row
+CONTEXT:  while executing command on localhost:57637
+-- a constraint that can be validated on all workers but one
+ALTER TABLE constrained_table
+  ADD CONSTRAINT failing_on_1_worker CHECK (constrained_column <> 125) NOT VALID;
+ALTER TABLE constrained_table
+  VALIDATE CONSTRAINT failing_on_1_worker;
+ERROR:  check constraint "failing_on_1_worker_8000014" is violated by some row
+CONTEXT:  while executing command on localhost:57638
+-- Check which constraints are validated
+SELECT *
+FROM constraint_validations
+ORDER BY 1, 2;
+       Constraint       | Validated? 
+------------------------+------------
+ failing_constraint     | f
+ failing_on_1_worker    | f
+ validatable_constraint | t
+(3 rows)
+
+SELECT *
+FROM constraint_validations_in_workers
+ORDER BY 1, 2;
+              name              | validated 
+--------------------------------+-----------
+ failing_constraint_8000009     | f
+ failing_constraint_8000010     | f
+ failing_constraint_8000011     | f
+ failing_constraint_8000012     | f
+ failing_constraint_8000013     | f
+ failing_constraint_8000014     | f
+ failing_constraint_8000015     | f
+ failing_constraint_8000016     | f
+ failing_on_1_worker_8000009    | f
+ failing_on_1_worker_8000010    | f
+ failing_on_1_worker_8000011    | f
+ failing_on_1_worker_8000012    | f
+ failing_on_1_worker_8000013    | f
+ failing_on_1_worker_8000014    | f
+ failing_on_1_worker_8000015    | f
+ failing_on_1_worker_8000016    | f
+ validatable_constraint_8000009 | t
+ validatable_constraint_8000010 | t
+ validatable_constraint_8000011 | t
+ validatable_constraint_8000012 | t
+ validatable_constraint_8000013 | t
+ validatable_constraint_8000014 | t
+ validatable_constraint_8000015 | t
+ validatable_constraint_8000016 | t
+(24 rows)
+
+DROP TABLE constrained_table;
+CREATE TABLE referenced_table (
+  id          int UNIQUE,
+  test_column int
+);
+SELECT create_distributed_table('referenced_table', 'id');
```

Comment:
Creating and removing the tables are costly in terms of the regression test execution times. Can't we use the same tables throughout the test?

Reply:
I made the necessary changes to reuse the 3 tables I created in the beginning of the tests, and drop them at the very end.